src/app/credExplorer/App.js

```diff
@@ -82,6 +82,14 @@ export function createApp(
       }
       return (
         <div style={{maxWidth: 900, margin: "0 auto", padding: "0 10px"}}>
+          <p>
+            <em>
+              This is a prototype. Please{" "}
+              <a href={process.env.SOURCECRED_FEEDBACK_URL}>
```

Comment:
Maybe add a test that the a has a href that is not null, just to verify there wasn't a typo or such